crates/prices/res/feeds.json

```diff
@@ -0,0 +1,54 @@
+{
+    "feeds": {
+        "mainnet": {
+            "USD": {
+                "1INCH": [
+                    ["chainlink", "0xc929ad75B72593967DE83E7F7Cda0493458261D9"]
+                ],
```

Comment:
I think something like this would make more sense:

```
1INCH: {
  chainlink: "0x....."
},
AAVE: {
  chainlink: "0x..."
}
```

this is probably why you weren't able to just derive the Deserializer

Reply:
that looks better, like I said previously I'm going to dive a deeper on the deserialization to get a better solution.

Reply:
I ended up using internal tags for derialization, let me know if you agree.